deepchem/models/torch_models/layers.py

```diff
@@ -10,10 +10,9 @@
     from torch import Tensor
     import torch.nn as nn
     import torch.nn.functional as F
-    from deepchem.models.torch_models.flows import Affine
 except ModuleNotFoundError:
     raise ImportError('These classes require PyTorch to be installed.')
-
+from deepchem.models.torch_models.flows import Affine
```

Comment:
This change shouldn't happen; the underlying issue may be due to using tqdm as a hard dependency